templates/zerver/integrations/codebase.md

```diff
@@ -1,18 +1,27 @@
+# Zulip Codebase integration
+
 Get Codebase notifications in Zulip!
 
-1. First, create the streams you’d like to use for Codebase notifications. There
-   will be two types of messages: commit-related updates and issue-related
-   updates. We recommend naming the streams `codebase` and `tickets`, respectively.
-   After creating these streams, make sure to subscribe all interested parties.
```

Comment:
This statement is implicit in channel creation, so removed it, and instead added a link to "create a channel".

Reply:
I don't see the link to the help center in your changes. Pushing  an update with that.